src-tauri/plugins/dfu/android/src/main/kotlin/com/betaflight/dfu/DfuPlugin.kt

```
@@ -0,0 +1,38 @@
+        super.load(webView)
+        bridge = DfuFdBridge(activity.applicationContext)
+        DfuNative.bind(bridge)
+        val app = activity.application
+        destroyCb = object : Application.ActivityLifecycleCallbacks {
+            override fun onActivityDestroyed(a: Activity) {
+                if (a !== activity) return
+                bridge.shutdown()
+                app.unregisterActivityLifecycleCallbacks(this)
+                destroyCb = null
+            }
+            override fun onActivityCreated(a: Activity, s: Bundle?) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)